test/integration/shared/lockup-linear/LockupLinear.t.sol

```diff
@@ -113,19 +113,4 @@ abstract contract LockupLinear_Integration_Shared_Test is Lockup_Integration_Sha
         params.totalAmount = totalAmount;
         streamId = lockupLinear.createWithTimestamps(params);
     }
-
-    /// @dev Creates the default stream with the provided sender and recipient.
-    function createDefaultStreamWithUsers(
-        address recipient,
-        address sender
-    )
-        internal
-        override
-        returns (uint256 streamId)
-    {
-        LockupLinear.CreateWithTimestamps memory params = _params.createWithTimestamps;
-        params.sender = sender;
-        params.recipient = recipient;
-        streamId = lockupLinear.createWithTimestamps(params);
-    }
```

Comment:
Why has this function been removed in this PR, @andreivladbrg?

Reply:
I think it got removed accidentally along with Hook test contracts. It has now been brought back in https://github.com/sablier-labs/v2-core/pull/873.

Reply:
Got it